roc/sender.go

```diff
@@ -1,9 +1,95 @@
 package roc
 
 /*
-#include <roc/sender.h>
+ #include <roc/sender.h>
+int roc_sender_write_frame(roc_sender* sender, float* samples, unsigned long samples_size) {
```

Comment:
Let's rename it to something like `rocGoSenderWriteFrame`, similar to https://github.com/roc-streaming/roc-go/blob/master/roc/log.c#L4

Or, better, `rocGoSenderWriteFloats`.